easy-digital-downloads.php

```diff
@@ -129,6 +129,14 @@ final class Easy_Digital_Downloads {
 	 */
 	public $cart;
 
+	/**
+	 * EDD Payment Stats Object
+	 *
+	 * @var object|EDD_Payment_Stats
```

Comment:
```suggestion
	 * @var EDD_Payment_Stats
```

It's always going to be `EDD_Payment_Stats` so we don't need the generic `object` in there. (I know the others have it, but they don't need it either.)